1001-2000/LIT1982Mashaf.xml

```diff
@@ -33,17 +33,39 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+         <abstract>
+            <p>Allegorical interpretation of a series of biblical passages from Genesis to Proverbs.</p>
+         </abstract>
+         <textClass>
+            <keywords>
+               <term key="Bible"/>
+               <term key="ChristianContent"/>
+               <term key="ChristianLiterature"/>
+               <term key="OldTestament"/>
+            </keywords>
+         </textClass>
+         <langUsage>
+            <language ident="en">English</language>
+            <language ident="gez">Gǝʿǝz</language>
+         </langUsage>
       </profileDesc>
       <revisionDesc>
          <change who="PL" when="2016-03-21"> Created file from google
             spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: text record</change>
+         <change who="CH" when="2023-04-16">Supplied abstract and incipit</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
-      <body>
-         <ab/>
-      </body>
+         <body>
+            <div xml:lang="gez" type="edition">
+               <note>The incipit is taken from <ref type="mss" corresp="BMLacq38#ms_i1"/></note>
+               <div type="textpart" subtype="incipit">
+                  <ab xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ። ወእምዝ፡ ንዌጥን፡ በረድኤተ፡ እግዚአብሔር፡ እንዘ፡ ንጽሕፍ፡ ዘንተ፡ መጽሐፈ፡ 
+                     ዘይሰመይ፡ መጽሐፈ፡ ጸሐይ። ወሶበ፡ ይበውእ፡ ውስተ፡ ልበ፡ ሰብእ፡ ያበርህ፡ ከመ፡ ፀሐይ። ይቤ፡ ኢሳይያስ፡ ወልደ፡ ትምህርቶሙ፡ ለቢዘናት። ቀዳሚ፡ ትንቢት፡ እምደብረ፡ 
+                     አራራት፡ ወአፈዋተ፡ ከርቤ፡ እም፡ አውግረ፡ ስኂን፡ ልዑላት፡ ዘውእቶሙ፡ ሐዋርያት። </ab>
+               </div>
```

Comment:
copy explicit already present in the issue as well
Explicit: እለኒ፡ ተኃቅፉ፡ እስራኤል፡ ቀዳሚ፡ ወይእዜሰ፡ ወጽኡ፡ እም፡ ታኅተ፡ ክንፊሁ፡ ለልዑል። እለኒ፡ ተኃጕሉ፡ እስራኤል፡ ወእለኒ፡ ተኃሠዩ፡ አሕዛብ። እለሰ፡ ተዓቀቡ፡ ቀዳሚ፡ እስራኤል፡ ወይእዜሰ፡ ወጽኡ፡ እምይብተራሁ፡ ወተማሠጥዎሙ፡ ፀላእቶሙ። ተመንደብኡ፡ በመርኃ፡ ብዙኅ።

Reply:
Done.